packages/core/src/api/schema/admin-api/api-key.api.graphql

```diff
@@ -0,0 +1,80 @@
+type Query {
+    apiKey(id: ID!): ApiKey
+    apiKeys(options: ApiKeyListOptions): ApiKeyList!
+}
+
+type Mutation {
+    """
+    Generates a new API-Key and attaches it to an Administrator.
+    Returns the generated API-Key.
+    API-Keys cannot be viewed again after creation.
+    """
+    createApiKey(input: CreateApiKeyInput!): CreateApiKeyResult!
+    "Updates an API-Key"
+    updateApiKey(input: UpdateApiKeyInput!): ApiKey!
+    "Deletes an API-Key"
+    deleteApiKey(input: DeleteApiKeyInput!): DeletionResponse!
+    """
+    Replaces the old with a new API-Key.
+    This is a convenience method to invalidate an API-Key without
+    deleting the underlying roles and permissions.
+    """
+    rotateApiKey(input: RotateApiKeyInput!): RotateApiKeyResult!
```

Comment:
Similarly to the deletion mutation, might wanna potentially change the input type to something similar to:

```graphql
rotateApiKeys(ids: [ID!]!): [RotateApiKeyResult!]!
```

Just thinking out loud, this'd lend itself more to bulk actions